grassroots-backend/src/providers/QueryRunnerProvider.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, InjectionToken, Provider } from "@nestjs/common";
+import { EntityTarget, ObjectLiteral, QueryRunner, Repository } from "typeorm";
+
+@Injectable()
+export class QueryRunnerProvider {
+  queryRunner: QueryRunner;
+
+  constructor(queryRunner: QueryRunner) {
+    this.queryRunner = queryRunner;
+  }
+
+  // This might be one level of abstraction too many, as often things will
```

Comment:
Curious on your take here.

Reply:
I'm not sure I understand this part of the PR. Maybe we could find a time this week to chat about it

Reply:
Sounds great! I'll write something up on this too, potentially to be added as a comment in this file at some point.